source/globalCommands.py

```diff
@@ -3288,6 +3293,9 @@ def script_toggleBrailleViewer(self, gesture: inputCore.InputGesture):
 		gesture="kb:NVDA+control+t"
 	)
 	def script_braille_toggleTether(self, gesture):
+		if config.conf["braille"]["mode"] == BrailleMode.SPEECH_OUTPUT.value:
+			ui.message(speechOutputMSG)
+			return

```

Comment:
Using the block action context, all these blocked functions can be marked as follows

```suggestion
	@gui.blockAction.when(gui.blockAction.Context.BRAILLE_MODE_SPEECH_OUTPUT)
	def script_braille_toggleTether(self, gesture):
```